configurations/rolling-upgrade-artifacts.yaml

```diff
@@ -0,0 +1,6 @@
+# workloads
+stress_before_upgrade: cassandra-stress write cl=QUORUM n=10000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=200 -pop seq=1..10000000 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5
```

Comment:
1. You can reduce it to be much less in term of dataset, currently it's 50GB, you can reduce the size of the cells or the number of partition or both. I think that 5GB should be fine (something that runs 1 minutes).

Reply:
Changed, now we spend 2-3 minutes on this